### PR TITLE
Redirects for Visual Studio 2015 tutorials

### LINKED ISSUE #1422: Redirect Visual Studio 2015 tutorials to Visual Studio 2017 versions.
The C# Getting Started topics for Visual Studio 2015 should all redirect to the Visual Studio 2017 version of the topic.

### PR DESCRIPTION
**Pull request by rpetrusha**
# Redirects for Visual Studio 2015 tutorials

## Summary

Visual Studio 2015 topics redirect to the corresponding Visual Studio 2017 topics.

Fixes #1422 

## Suggested Reviewers

@mairaw

### CODE REVIEW

**Comment by mairaw** [comment]
the build has warnings. can you take a look?

**Comment by rpetrusha** [comment]
@mairaw A bad link, which I've fixed.

File: docs/core/tutorials/using-on-windows.md [hunk 1]
```diff
@@ -1,270 +1,10 @@
...
-  "dependencies": {
-    "NETStandard.Library": "1.6.0",
-    "Microsoft.NETCore.Portable.Compatibility": "1.0.1"
-  },
-  "frameworks": {
-    "netstandard1.3" : {}
-  }
-}
-```
+## Content updated!
```
Comment (mairaw) [inline docs/core/tutorials/using-on-windows.md]: should we say retired, removed or archived instead of updated?

**Comment by rpetrusha** [comment]
I like "retired" better. I'll update the topics.

**Comment by mairaw** [comment]
LGTM! :shipit:

**Pull request merged**